src/Entity/EntityIdValue.php

```diff
@@ -31,8 +31,8 @@ public function __construct( EntityId $entityId ) {
 	 */
 	public function serialize() {
 		return json_encode( [
-			$this->entityId->getEntityType(),
-			$this->getNumericId()
+			get_class( $this->entityId ),
```

Comment:
It's correct that these `serialize` formats are not used in our JSON encoding, but they are used to build summaries.